addons/website_sale/data/demo.xml

```diff
@@ -643,19 +642,26 @@
             <field name="uom_id" ref="uom.product_uom_unit"/>
             <field name="uom_po_id" ref="uom.product_uom_unit"/>
             <field name="description_sale">Warranty, issued to the purchaser of an article by its manufacturer, promising to repair or replace it if necessary within a specified period of time.</field>
-            <field name="attribute_value_ids" eval="[(6,0,[ref('product.product_attribute_value_5')])]"/>
             <field name="categ_id" ref="product.product_category_3"/>
             <field name="invoice_policy">delivery</field>
+            <field name="public_categ_ids" eval="[(6,0,[ref('website_sale.services')])]"/>
         </record>
-        <record id="product_template_attribute_line_1" model="product.template.attribute.line">
-            <field name="product_tmpl_id" ref="product_product_1_product_template"/>
+
+        <record id="website_sale.product_template_attribute_line_1" model="product.template.attribute.line">
```

Comment:
why prefix with current module ? idem for all others

Reply:
Because it is confusing when two id appear to be the same because they have the same name, but in reality they are different records because they have been defined in different modules. It's harder to see what is a new record and what is an override without the prefix.

Reply:
Justement ... quand on le préfix par un nom de module c'est Pour l’override généralement ! Sinon on utilise la notation « relative » qui prend le nom du module courant !